packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -385,99 +365,36 @@ export const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModal
                   />
                 ) : null}
                 {fields?.fingerprint ? (
-                  <>
-                    {!isCertificateQueryEnabled ? (
+                  <ValidatedTextInput
+                  field={fields.fingerprint}
+                  isRequired
+                  fieldId="fingerprint"
+                  formGroupProps={{
+                    labelIcon: (
+                      <Popover
+                        bodyContent={
+                          <div>
+                            If your vSphere environment uses untrusted, self-signed certificates to authenticate connections,
+                            you must specify the <code>thumbprint</code> of the vCenter Server 
+                            If your vSphere environment uses trusted certificates that are signed by a known Certificate Authority (CA),
+                            you do not need to specify the <code>thumbprint</code> option.
```

Comment:
Is this still correct?  Will the backend go and lookup the certificate and calcualte the thumbprint if the certificate can be verified by a known CA?

Reply:
> Is this still correct?

yes, the backend need a fingerprint, it does not need ca-cert
see: https://konveyor.github.io/forklift/migratingvms/migratecli/

> Will the backend go and lookup the certificate and calculate the thumbprint if the certificate can be verified by a known CA?

yes, the backend will not require ca or fingerprint if the connection is using trusted cert, when using self sign vmware only needs the fingerprint
see: https://docs.vmware.com/en/VMware-vSphere-Bitfusion/3.0/Install-Guide/GUID-00D07073-5C20-4E78-B5C9-8A79610E73E8.html

Reply:
but since we make this field required, maybe I should not talk about 

> If your vSphere environment uses trusted certificates that are signed by a known Certificate Authority (CA),
> you do not need to specify the <code>thumbprint</code> option."

I'll remove it

Reply:
yes, we talked about it in todays standup, the provider is doing some calls to get vddk image using the thumbprint even if you ask for o/w insecure connection, to the provider requires the thumbprint always :-(

not sure if we need to explain that in the help text, i opted to keep it simple, and just make it a required field

Reply:
I removed the part about trusted environment not needing the thumbprint because it's required field , didn't add an explanation about our lacking implementation of fetching the vddk image ending up needing the thumbprint always.